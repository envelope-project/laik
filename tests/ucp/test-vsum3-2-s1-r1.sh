#!/bin/sh
timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
export UCP_COMMANDS_FILE=/home/ge96hoy2/laik/tests/ucp/test-vsum3-2-s1-r1-commands.txt
timeout 2 ./ucprun.sh --strict-node 0 -n 2 -s 1 ../../examples/vsum3 1 | LC_ALL='C' sort > test-vsum3-2-s1-r1.out
cmp test-vsum3-2-s1-r1.out "$(dirname -- "${0}")/test-vsum3-2-s1-r1.expected"
