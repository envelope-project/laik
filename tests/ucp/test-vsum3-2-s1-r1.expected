Phase 0, Epoch 0, Proc 0/2: sum of 5000 values at 0 - 4999 : 12497500
Phase 0, Epoch 0, Proc 1/2: sum of 5000 values at 5000 - 9999 : 37497500
Phase 0: total sum: 49995000
Phase 1, Epoch 1, Proc 0/2: sum of 5000 values at 0 - 4999 : 12497500
Phase 1, Epoch 1, Proc 1/2: sum of 5000 values at 5000 - 9999 : 37497500
Phase 1: total sum: 49995000
